micrometer-core/src/test/groovy/io/micronaut/configuration/metrics/management/endpoint/MetricsEndpointSpec.groovy

```diff
@@ -190,7 +190,7 @@ class MetricsEndpointSpec extends Specification {
 
 
     @Unroll
-    void "test metrics endpoint get jvm details #name success with tags"() {
+    void "test metrics endpoint get jvm details success with tags"() {
```

Comment:
Is this change necessary?

Reply:
Well, there isn't a `name` unroll variable, and the report was displaying `test metrics endpoint get jvm details #name success with tags`. It is a leftover from a copy/paste from other test.

But strictly speaking, it is not necessary